test/utilities/setRunsTimestamps.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+const { sequelize, repositories: { RunRepository } } = require('../../lib/database');
+
+const moduleSetupTime = Date.now();
+
+/**
+ * Created custom escaped timestamp string for test purpose
+ * @param {Number} runNumber run_number
+ * @returns {String} escaped date formated for mariadb
+ */
+const getEscapedPastDate = (runNumber) => `'${new Date(moduleSetupTime - runNumber * 1000 * 60 * 30)
+    .toISOString()
+    .replace(/T/, ' ')
+    .replace(/\..+/, '')}'`;
+
+/**
+ * This var holds run_numbers which will have customized timestamps
+ */
+const runRunNumbersWithCustomizedTimestamps = [1, 2, 3, 4];
+
+/**
+ * Set timestamps (created_at and updated_at) for runs with run_number specified in variable @see runRunNumbersWithCustomizedTimestamps
+ * according to formula `UPSERTED_TIMESTAMP = THIS_METHOD_CALL_TIME - run_number * 1000 * 60 * 30` (run_number * 30min)
+ * so there are time differences between the runs
+ * @return {Promise<Array<Object>>} information from db whether update was successful or not
+ */
+const setRunsTimestamps = async () => {
```

Comment:
When you name function, think that this function name will be suggested in auto completion. 
`setRunsTimestamp` is way too generic to be exported as is, especially when it's a test function.